OpenRA.Mods.Common/Traits/Voiced.cs

```diff
@@ -27,19 +27,20 @@ public class VoicedInfo : TraitInfo
 		public override object Create(ActorInitializer init) { return new Voiced(this); }
 	}
 
-	public class Voiced : IVoiced
+	public class Voiced : ConditionalTrait<VoicedInfo>, IVoiced
 	{
-		public readonly VoicedInfo Info;
-
 		public Voiced(VoicedInfo info)
+			: base(info)
```

Comment:
```suggestion
		public Voiced(VoicedInfo info)
			: base(info) { }
```

and remove the braces below.